File: Typescript/Practice/medium/medium_howMuchTrue.py
#!/usr/bin/env python

def count_true(list1):
  counter = 0
  for i in list1:
    if i == True:
      counter += 1
  return counter

print(count_true([True, False, False, True, False]))
print(count_true([False, False, False, False]))
print(count_true([]))

File: Typescript/Practice/medium/medium_howMuchTrue.ts
#!/usr/bin/env npx ts-node

function countTrue (arr: Array<boolean>): number {
  let num: number = 0;
  for (let i of arr) {
    if (i === true) {
      num++
    };
  };
  return num;
};

console.log(countTrue([true, false, false, true, false]));
console.log(countTrue([false, false, false, false]));
console.log(countTrue([]));


